src/CommandAndStateMessageParser.hpp

```diff
@@ -85,6 +86,7 @@ namespace deep_trekker {
         Json::Value mJData;
         Json::CharReaderBuilder mRBuilder;
         std::unique_ptr<Json::CharReader> mReader;
+        double m_camera_head_tilt_position_command;
```

Comment:
This should be initialized to `base::unknown<double>()` to avoid using an unknown value.